src/MarginBase.sol

```diff
@@ -625,55 +561,61 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
     }
 
     /// @notice register a limit order internally and with gelato
-    /// @param _marketKey: synthetix futures market id/key
+    /// @param _marketKey: Synthetix futures market id/key
     /// @param _marginDelta: amount of margin (in sUSD) to deposit or withdraw
     /// @param _sizeDelta: denominated in market currency (i.e. ETH, BTC, etc), size of futures position
     /// @param _targetPrice: expected limit order price
     /// @param _orderType: expected order type enum where 0 = LIMIT, 1 = STOP, etc..
+    /// @param _priceImpactDelta: price impact tolerance as a percentage
     /// @return orderId contract interface
     function placeOrder(
         bytes32 _marketKey,
         int256 _marginDelta,
         int256 _sizeDelta,
         uint256 _targetPrice,
-        OrderTypes _orderType
+        OrderTypes _orderType,
+        uint128 _priceImpactDelta
     ) external payable returns (uint256) {
         return
-            _placeOrder(
-                _marketKey,
-                _marginDelta,
-                _sizeDelta,
-                _targetPrice,
-                _orderType,
-                0
-            );
+            _placeOrder({
+                _marketKey: _marketKey,
+                _marginDelta: _marginDelta,
+                _sizeDelta: _sizeDelta,
+                _targetPrice: _targetPrice,
+                _orderType: _orderType,
+                _maxDynamicFee: 0,
+                _priceImpactDelta: _priceImpactDelta
+            });
     }
 
     /// @notice register a limit order internally and with gelato
-    /// @param _marketKey: synthetix futures market id/key
+    /// @param _marketKey: Synthetix futures market id/key
     /// @param _marginDelta: amount of margin (in sUSD) to deposit or withdraw
     /// @param _sizeDelta: denominated in market currency (i.e. ETH, BTC, etc), size of futures position
     /// @param _targetPrice: expected limit order price
     /// @param _orderType: expected order type enum where 0 = LIMIT, 1 = STOP, etc..
     /// @param _maxDynamicFee: dynamic fee cap in 18 decimal form; 0 for no cap
+    /// @param _priceImpactDelta: price impact tolerance as a percentage
     /// @return orderId contract interface
     function placeOrderWithFeeCap(
         bytes32 _marketKey,
         int256 _marginDelta,
         int256 _sizeDelta,
         uint256 _targetPrice,
         OrderTypes _orderType,
-        uint256 _maxDynamicFee
+        uint256 _maxDynamicFee,
+        uint128 _priceImpactDelta
```

Comment:
Would be better to have "optional" params at the end

Reply:
done!